controllers/rest/controller_standard.go

```diff
@@ -665,6 +665,105 @@ func DeleteController(api *RESTAPI, projection projections.Projection, commandDi
 	}
 }
 
+//DefaultResponseMiddleware returns content type based on content type in example
+func DefaultResponseMiddleware(api *RESTAPI, projection projections.Projection, commandDispatcher model.CommandDispatcher, eventSource model.EventRepository, entityFactory model.EntityFactory, path *openapi3.PathItem, operation *openapi3.Operation) echo.MiddlewareFunc {
+
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		return func(ctxt echo.Context) error {
+			ctx := ctxt.Request().Context()
+			//take media type from the request since the context wouldnt add it because there is no entity factory to use
+			mediaType := ctxt.Request().Header.Get(weoscontext.ACCEPT)
+			var bytesArray []byte
+			var err error
+			contentType := ""
+			var respCode int
+			found := false
+			for code, resp := range operation.Responses {
+				if found {
+					break
+				}
+				respCode, _ = strconv.Atoi(code)
+				if resp.Value.Content != nil {
```

Comment:
Sending this header is optional. I think the code here should be written in a way that it only tries to loop through and find a matching type IF the Accept header is passed in AND there are multiple responses. If there is not response that matches the Accept header you should server the first one (Accept header is a HINT not a requirement https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation)